src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,85 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+import { Person } from '../types';
+import { PersonLink } from '../components/PersonLink';
+import { SearchLink } from './SearchLink';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+
+  const sortType = searchParams.get('sort');
+  const sortOrder = searchParams.get('order');
+
+  const findParentPerson = (parentName: string) => {
+    return people.find(person => {
+      return person.name === parentName;
+    });
+  };
```

Comment:
This `findParentPerson` function might be inefficient if the `people` array is large because it searches through the entire array every time it's called. It might be better to create a map of person names to people at the start of the component and use that for lookups instead.